sdks/python/apache_beam/transforms/core.py

```diff
@@ -2598,8 +2598,9 @@ def typed(transform):
 
       def inject_default(_, combined):
         if combined:
-          assert len(combined) == 1
-          return combined[0]
+          if len(combined) > 1:
+            _LOGGER.warning('Apply Combined Fn with this list: %s', combined)
+            return combined[-1]
```

Comment:
We cannot safely just "drop" extra values, giving back an arbitrary, uncombined choice.

Reply:
This is just my test. :)
Restored the code but added some error messages to track this.